examples/chart/teleport-kube-agent/values.yaml

```diff
@@ -64,6 +64,8 @@ labels: {}
 # Container image for the agent. Since this runs without the auth_service, we
 # don't need the enterprise version.
 image: quay.io/gravitational/teleport
+# Optional secret to use when pulling image from a private registry
+imagePullSecret: ""
```

Comment:
Could you restructure this similar to https://github.com/gravitational/teleport/blob/ab02b4dad8aea8a92c34820bbe254e0c01073e56/examples/chart/teleport/values.yaml#L20-L21 ?

Reply:
fix: https://github.com/gravitational/teleport/pull/6941/commits/8e11fb6f669e3dc6e25e89436efc874c1146999e